src/go/rdctl/pkg/plist/plist.go

```diff
@@ -25,46 +25,59 @@ import (
 const indentChange = "  "
 
 // convertToPListLines recursively reflects the supplied value into lines for a plist
-func convertToPListLines(v reflect.Value, indent string) ([]string, error) {
-	kind := v.Kind()
+func convertToPListLines(typedValue reflect.Value, actualValue reflect.Value, indent string) ([]string, error) {
+	kind := typedValue.Kind()
+	if actualValue.Kind() == reflect.Interface && typedValue.Kind() != reflect.Interface {
+		if actualValue.IsNil() {
+			return nil, nil
+		}
+		return convertToPListLines(typedValue, actualValue.Elem(), indent)
+	}
+	if actualValue.Kind() == reflect.Ptr {
+		return nil, fmt.Errorf("plist generation: got an unexpected pointer for value %v, typedValue %v\n", actualValue, typedValue)
```

Comment:
My linter complains about this ending with a new line for some reason.

Reply:
Your linter is correct ... `fmt.Errorf` strings normally shouldn't end with a newline.